src/style.css

```diff
@@ -1 +1,80 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+.nav__list,
```

Comment:
```suggestion
ul,
```

Reply:
Why p2 in checklist doesn't apply to this situation?

Reply:
I mean to reset styles we use tags, as u did on the lesson